jni/ooni_test_wrapper.hpp

```diff
@@ -0,0 +1,52 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+#ifndef JNI_OONI_TEST_WRAPPER_HPP
+#define JNI_OONI_TEST_WRAPPER_HPP
+
+#include <measurement_kit/ooni.hpp>
+#include <string>
+
+class OoniTestWrapper {
+  public:
+    OoniTestWrapper(std::string test_name) {
+        if (test_name == "tcp_connect") {
+            real_test_.reset(new mk::ooni::TcpConnect);
+        } else {
+            throw std::runtime_error("invalid test name");
+        }
+    };
+
+    void set_verbosity(uint32_t verbosity) {
+        real_test_->set_verbosity(verbosity);
+    }
+
+    void increase_verbosity() {
+        real_test_->increase_verbosity();
+    }
+
+    void set_input_filepath(std::string fpath) {
+        real_test_->set_input_filepath(fpath);
+    }
+
+    void set_output_filepath(std::string fpath) {
+        real_test_->set_output_filepath(fpath);
+    }
+
+    void set_error_filepath(std::string fpath) {
+        // XXX: this requires MK v0.3.0
```

Comment:
@lorenzoPrimi this cannot be implemented until we start using 0.3.0 (and the same holds for the web_connectivity test, which suggests that I'd better tag an alpha release asap).